src/components/Repository.js

```diff
@@ -26,7 +26,7 @@ function Repository({user, match}) {
   } = match;
   const [repository, setRepository] = useState(null);
   const [error, setError] = useState(null);
-  const [note, setNote] = useState(location.note);
+  const [note, setNote] = useState(null);
```

Comment:
Curious about this as well. Does this solve an error that was overlooked?

Reply:
I’ve made this adjustment offline in the vite branch in response to some errors. but haven’t committed or pushed. I don’t know what `location.note` Referred to but being that it was the prefetch state value something “empty” makes sense to have here.

Reply:
`location.note` was never really populated - that key doesn't exist in the location object